### PR TITLE
Fix scroll upshot in voila rendering context, bump lumino dependencies

### PR DESCRIPTION
**Pull request by ibdafna**
Signed-off-by: Itay Dafna <i.b.dafna@gmail.com>

### CODE REVIEW

**Review by gaborbernat — DISMISSED** [review]

**Comment by ibdafna** [comment]
@gaborbernat I saw some Python test suite failures on versions >3.6 and re-triggered the tests. I can't reproduce locally. Also this code change should not affect the Python side.

**Comment by ibdafna** [comment]
@martinRenou looks related to your PR in jupyter_client:

https://github.com/jupyter/jupyter_client/commit/1363f53f308f0d3621950dcbd2e49edd89ab4aa6

**Comment by ibdafna** [comment]
EDIT: I can now reproduce locally with a fresh environment and python=3.9.

**Comment by ibdafna** [comment]
Discussed this with @martinRenou. We will fix the serialisation issue in `ipywidgets`, and in the meantime I will pin `jupyter_client` to version 6.2.0.

**Comment by SylvainCorlay** [comment]
> Discussed this with @martinRenou. We will fix the serialisation issue in ipywidgets, and in the meantime I will pin jupyter_client to version 6.2.0.

@ibdafna are you still around?

**Comment by SylvainCorlay** [comment]
It is probably not a good idea to support the case of tuple keys in the js serialization of ipykernel.

**Comment by ibdafna** [comment]
@SylvainCorlay still around. I don't think there's a way around that for serialisation of nested-index pandas DataFrames. Why is it a bad idea? Tuples are immutable, and they can also be converted to string if necessary.

**Comment by SylvainCorlay** [comment]
> @SylvainCorlay still around. I don't think there's a way around that for serialisation of nested-index pandas DataFrames. Why is it a bad idea? Tuples are immutable, and they can also be converted to string if necessary.

Walking over all JSON entries in ipykernel to find tuple keys is very inefficient.
It should probably be converted to string in the client code.

**Review by gaborbernat — DISMISSED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -4,6 +4,7 @@ requires = [
     "wheel>=0.30",
     "jupyter_packaging~=0.9.0",
     "jupyterlab~=3.0",
+    "jupyter_client~=6.2.0"
```
Comment (martinRenou) [inline pyproject.toml:7]: This shouldn't be needed if https://github.com/bloomberg/ipydatagrid/pull/274 gets in

**Review by martinRenou — CHANGES_REQUESTED** [review]

**Review by kaiayoung — APPROVED** [review]

**Pull request merged**